distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1688,6 +1901,62 @@ private CometDClient setupCometDClient(List<String> channels) throws Exception {
         return cometDClient;
     }
 
+    /**
+     * Used to compare the expected information about the download and the actual information
+     * pulled from the list about a download.
+     * @param download the map containing the actual download list information to compare.
+     * @param downloadId the expected download id as a string to be compared with the actual.
+     * @param filename the expected filename as a string to be compared with the actual.
+     * @param resourceData the expected resource data as a string to be compared with the actual.
+     */
+    private void checkDownloadInProgress(Map download, String downloadId, String filename,
+            String resourceData) {
+        assertThat(download.get("downloadId"), is(downloadId));
+        assertThat(download.get("fileName"), is(filename));
+        assertThat(download.get("status"), is("IN_PROGRESS"));
+        int bytesDownloaded = (int) download.get("bytesDownloaded");
+        assertThat(bytesDownloaded, is(greaterThan(0)));
+        assertThat(bytesDownloaded, is(lessThan(resourceData.length() + 1)));
+        assertTrue(((String) download.get("percentDownloaded")).matches(
+                "UNKNOWN|[0-9]|[0-9]{2}|100"));
+        assertThat((List<String>) download.get("users"), contains("Guest@Guest@127.0.0.1"));
+
+    }
+
+    /**
+     * Sets up what the response of the csw server will be when a resource is requested for download.
+     * @param metacardId the metacardId of the resource that may be requested from the csw server.
+     * @param filename the filename of the resource that may be requested from the csw server.
+     * @param resourceData the resource data of the resource that may be requested from the csw server.
+     * @param retries the number of times the server will indicate that it retried downloading.
+     * @param delayBetweenRetries the number of milliseconds between each download retry.
```

Comment:
We should rename `delayBetweenRetries` to `delayBetweenChunks` and update the comment.